jax/interpreters/pxla.py

```diff
@@ -836,10 +843,26 @@ def execute_replicated(compiled, backend, in_handler, out_handler, *args):
 xla_pmap_p.def_custom_bind(xla_pmap)
 xla_pmap_p.def_impl(xla_pmap_impl)
 
-def _pmap_translation_rule(c, axis_env,
-                           in_nodes, name_stack, axis_name, axis_size,
-                           global_axis_size, devices, name,
-                           call_jaxpr, *, backend=None, mapped_invars):
+def _pmap_translation_rule(
```

Comment:
Every time you use an unnecessary newline character @mattjj dies a little inside...

(This is fine, I jest... but we should all pitch in and buy him a larger monitor.)

Reply:
Guys I just have my 13" laptop at home, I'm dying.

Reply:
Honestly I love this PR, EXCEPT for these super tall argument lists :P

Reply:
Heh I have a different opinion 😄 . Have reverted this (and all other signatures) to the current "condensed and lightly typed" style matching the rest of the file.